assets/css/main.css

```diff
@@ -1,11 +1,8864 @@
-﻿/*!
+@charset "UTF-8";
```

Comment:
This file is supposed to be minified CSS and written by the `drone exec` pipeline (building Sass). Something has gone wrong in your build since this is not minified. Run `drone exec` again.

Reply:
Yeah, I don't know what happened. It is minified now.